packages/e2e/features/ui/hooks/actions-navigation.feature

```diff
@@ -8,15 +8,15 @@ Feature: Navigate action hooks
 
   @react
   Scenario: Navigation anchor action works correctly
-    When I click the "hash" button
+    When I click the "Go to #notes" button
     Then My url contains "#notes"
 
   @react
   Scenario: Navigation reload action works correctly
-    When I click the "reload" button
+    When I click the "Reload" button
     Then My page should be reloaded
 
   @react
   Scenario: Navigation location change action works correctly
-    When I click the "locationChange" button
+    When I click the "Go to amazon.com" button
```

Comment:
These tests now use accessible role queries instead of test IDs.

Reply:
There was a reason were using test IDs here though. These are testing the three different types of actions whereas not is says nothing about the types of navigations being tested. If there's a better way to do this, I'm all ears, but I think this makes the test less clear now.

Reply:
The `Scenario` (the name of the test) blocks describe which action is being tested. I'd say that is more communicative than a test ID like "hash". What do you think?